src/autocomplete/shared/tokenize.ts

```diff
@@ -41,7 +43,9 @@ export function tokenize<L extends LexerType>(
             if (tokenName) {
                 tokens.push({
                     ruleName: tokenName,
-                    startIndex: token.column,
+                    startIndex: token.start,
+                    line: token.line,
+                    column: token.column + 1,
```

Comment:
@itwillwork `startIndex` is invalid naming for column index, I made `column` field, which contain column position.
I can make `columnIndex` for you if you want

Reply:
I think it will be better to return column as it is in token interface and then do the transformation inside extractPostgreSqlStatementsFromQuery function

Reply:
True, this will break the code